CONTRIBUTING.md

```diff
@@ -1,6 +1,11 @@
 ## Contributing
 
-Chat with us in the [bugbug](https://chat.mozilla.org/#/room/#bugbug:mozilla.org) Matrix room.
-
-1. [Issues marked as `good-first-bug`](https://github.com/mozilla/bugbug/labels/good-first-bug) are self-contained enough that a contributor should be able to work on them.
-2. Issues are considered not assigned, until there is a PR linked to them. Feel free to work on any unassigned issue, you don't need to ask first.
+Steps to contribute a project on Github for beginners:
```

Comment:
I have made a series of suggestions below about what can be in the file, please go through each suggestion and try to make the commands appear as they are (using markdown syntax)so that contributors can easily copy and paste. Enjoy and thanks @srishtig2412 :)